services/movie.py

```diff
@@ -28,13 +33,14 @@ def create_movie(
     genres_ids: list = None,
     actors_ids: list = None,
 ) -> Movie:
-    movie = Movie.objects.create(
-        title=movie_title,
-        description=movie_description,
-    )
-    if genres_ids:
-        movie.genres.set(genres_ids)
-    if actors_ids:
-        movie.actors.set(actors_ids)
-
-    return movie
+    with transaction.atomic():
+        movie = Movie.objects.create(
```

Comment:
use transaction.atomic decorator. The decorator is a more concise and cleaner way to ensure that the function executes within a transaction.